coral/src/app/layout/SidebarNavigation.test.tsx

```diff
@@ -0,0 +1,63 @@
+import SidebarNavigation from "src/app/layout/SidebarNavigation";
+import { cleanup, screen, render, within } from "@testing-library/react";
+
+// mock out svgs to avoid clutter
+jest.mock("@aivenio/design-system", () => {
+  return {
+    __esModule: true,
+    ...jest.requireActual("@aivenio/design-system"),
+
+    Icon: () => {
+      return <div data-testid={"ds-icon"}></div>;
+    },
+  };
+});
+
+// "http://localhost/" comes from window.location.origin
+// that represents the Angular app
+const navLinks = [
+  {
+    name: "Overview",
+    linkTo: "http://localhost/index",
+  },
+  { name: "Topics", linkTo: "/" },
+  { name: "Kafka Connector", linkTo: "http://localhost/kafkaConnectors" },
+  { name: "Schemas", linkTo: "/" },
+  { name: "User and teams", linkTo: "/" },
+  { name: "Audit log", linkTo: "/" },
+  { name: "Settings", linkTo: "/" },
+];
+
+describe("SidebarNavigation.tsx", () => {
+  beforeAll(() => {
+    render(<SidebarNavigation />);
+  });
+
+  afterAll(cleanup);
+
+  it("renders the primary navigation", () => {
+    const nav = screen.getByRole("navigation", { name: "Primary navigation" });
+    expect(nav).toBeVisible();
+  });
+
+  navLinks.forEach((link) => {
```

Comment:
This is an excellent way of making sure a set of navigation items are present. However, we could also add a test, that checks that there are not extras.

`it("has correct amount of navigation items", () => {...)`

And the expect could be kid of similar as in here:
```
const nav = screen.getByRole("navigation", {
  name: "Primary navigation",
});

const icons = within(nav).getAllByTestId("ds-icon");
expect(icons).toHaveLength(navLinks.length);
```

Reply:
Yes, that's a great addition to the tests! ✨